### PR TITLE
[RM Ch7.2] Attacker behaviour table

### PR DESCRIPTION
**Pull request by walterkozlowski**
Added typical cloud attacker table behaviour

### CODE REVIEW

File: doc/ref_model/chapters/chapter07.md [hunk 1]
```diff
@@ -59,6 +59,19 @@ This chapter examines multiple aspects of security as it relates to Cloud Infras
...
 
+Typical cloud associated attacker behaviours, as identified in the widely accepted MITRE ATT&CK® Framework https://www.mitre.org/sites/default/files/publications/mitre-getting-started-with-attack-october-2019.pdf. This framework provides a systematic approach to capture adversarial behaviour targeting cloud environments. Examples of such adversarial behaviours are listed in the table below.
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter07.md:62]:
```suggestion
Typical cloud associated attacker behaviours, as identified in the widely accepted [MITRE ATT&CK® Framework](https://www.mitre.org/sites/default/files/publications/mitre-getting-started-with-attack-october-2019.pdf). This framework provides a systematic approach to capture adversarial behaviours targeting cloud environments. Examples of such adversarial behaviours are listed in the table below.
```

File: doc/ref_model/chapters/chapter07.md [hunk 2]
```diff
@@ -59,6 +59,19 @@ This chapter examines multiple aspects of security as it relates to Cloud Infras
...
+ 
+ |  Attacker behaviour | Description  | 
+ |-------|------|
+ |Initial Access|Compromising user administration accounts that are not protected by multi-factor authentication|
+ |Evasion|Modifying cloud compute instances in the production environment by modifying virtual instances for attack staging|
+ |Discovery|Using open-source tools to discover what cloud services are operating and then disabling them in a later stage to avoid detection|
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter07.md:68]: what  is "disabled"? If just "tools" then logs would still exist

File: doc/ref_model/chapters/chapter07.md [hunk 3]
```diff
@@ -59,6 +59,19 @@ This chapter examines multiple aspects of security as it relates to Cloud Infras
...
+ |Data Exfiltration|Moving data from the customer’s production databases to the hacker’s cloud service account or transferring the data out of the Communication Service Provider (CSP) to the attacker’s private network|
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter07.md:69]: Hackers have customers -- I suppose that is why "customers" pay a ransom.

Can we change to "compromised production databases" or "under attack production databases"?

File: doc/ref_model/chapters/chapter07.md [hunk 4]
```diff
@@ -59,6 +59,19 @@ This chapter examines multiple aspects of security as it relates to Cloud Infras
...
+ |Service Impact|Creating denial-of-service availability issues by modifying Web Application Firewall (WAF) rules and compromising APIs and web-based GUIs|
+
+<p align="center"><b>Table 7-0:</b> Cloud attacker bahaviours</p>
```
Comment (karinesevilla) [inline doc/ref_model/chapters/chapter07.md:72]: The title of the table must reflect that these are examples of attacks

File: doc/ref_model/chapters/chapter07.md (hunk 2 above)
Comment (walterkozlowski) [reply doc/ref_model/chapters/chapter07.md:68]: @pgoyal01: I am not sure what you mean by your comment.  Logs can still exist but may not be monitored or actioned, or they may look like legitimate use of discovery tools.  Happy to discuss in the RM meeting, as I might be missing something.

File: doc/ref_model/chapters/chapter07.md [hunk 5]
```diff
@@ -59,6 +59,19 @@ This chapter examines multiple aspects of security as it relates to Cloud Infras
...
 
+Typical cloud associated attacker behaviours, as identified in the widely accepted [MITRE ATT&CK® Framework](https://www.mitre.org/sites/default/files/publications/mitre-getting-started-with-attack-october-2019.pdf). This framework provides a systematic approach to capture adversarial behaviours targeting cloud environments. Examples of such adversarial behaviours are listed in the table below.
+ 
+ |  Attacker behaviour | Description  |
```
[the remark this answers is not in the document]
Comment (walterkozlowski) [reply doc/ref_model/chapters/chapter07.md]: Changed, please check, @karinesevilla

File: doc/ref_model/chapters/chapter07.md (hunk 4 above)
Comment (walterkozlowski) [reply doc/ref_model/chapters/chapter07.md]: changed, please check: @karinesevilla

**Review by karinesevilla — APPROVED** [review]

**Review by pgoyal01 — APPROVED** [review]

File: doc/ref_model/chapters/chapter07.md [hunk 6]
```diff
@@ -59,6 +59,19 @@ This chapter examines multiple aspects of security as it relates to Cloud Infras
...
 
+Typical cloud associated attacker tactics have been identified in the widely accepted [MITRE ATT&CK® Framework](https://www.mitre.org/sites/default/files/publications/mitre-getting-started-with-attack-october-2019.pdf). This framework provides a systematic approach to capture adversarial tactics targeting cloud environments. Examples of such adversarial tactics are listed in the table below.
+ 
+ |  Attacker tactics | Examples  | 
+ |-------|------|
+ |Initial Access|Compromising user administration accounts that are not protected by multi-factor authentication|
+ |Evasion|Modifying cloud compute instances in the production environment by modifying virtual instances for attack staging|
+ |Discovery|Using open-source tools to discover what cloud services are operating and then disabling them in a later stage|
```
Comment (ildikov) [inline doc/ref_model/chapters/chapter07.md:68]: Do we need to differentiate whether the attacker discovers the cloud services with 'open-source' tools?

I would remove that and just say "Discovering what cloud services..."

Comment (walterkozlowski) [reply doc/ref_model/chapters/chapter07.md:68]: @ildikov: good comment, thanks! I am changing it that way.  Please review and approve if you can.

**Comment by walterkozlowski** [comment]
@ildikov, proposed change done.  Thank you for feedback!

**Review by ildikov — APPROVED** [review]
Thanks Walter!

**Pull request merged**